tests/JavaTest.java

```diff
@@ -377,11 +383,17 @@ static void TestBuilderBasics(FlatBufferBuilder fbb, boolean sizePrefix) {
         TestEq(monster.testarrayoftables(0).name(), "Barney");
         TestEq(monster.testarrayoftables(1).name(), "Frodo");
         TestEq(monster.testarrayoftables(2).name(), "Wilma");
+        TestEq(monster.testarrayoftablesVector().get(0).name(), "Barney");
```

Comment:
maybe get the vector just once, to not promote people to copy this style? :)

Reply:
Fixed :-)